jdaviz/components/tooltip.vue

```diff
@@ -66,7 +66,9 @@ const tooltips = {
   'viewer-data-radio': 'Switch visibility to layers associated with this data entry',
   'viewer-data-enable': 'Load data entry into this viewer',
   'viewer-data-disable': 'Disable data within this viewer (will be hidden and unavailable from plugins until re-enabled)',
+  'viewer-wcs-delete': 'Remove orientation option across entire app',
   'viewer-data-delete': 'Remove data entry across entire app (might affect existing subsets)',
+  'viewer-data-nowcs': 'Data does not have WCS, cannot add unless link type changed to pixel',
```

Comment:
Except this -- This check blocks a markers table from being loaded in the new viewer. This table was added with SkyCoord data and linked to the data by RA and Dec. But it does not have spatial WCS because it is a table.

![Screenshot 2023-11-02 160953](https://github.com/bmorris3/jdaviz/assets/2090236/426f1054-6270-4fd1-afe7-09d591b5307c)

Reply:
ok, then can you propose changes to the `has_wcs` line where I [commented here](https://github.com/bmorris3/jdaviz/pull/14#discussion_r1380619041) so that the data-menu sees this as being eligible when linked by WCS?  (And maybe we would then want to rename that variable to `supports_sky` or something)